src/renderer/store/settings.store.ts

```diff
@@ -282,6 +283,7 @@ const initialState: SettingsState = {
     },
     general: {
         accent: 'rgb(53, 116, 252)',
+        buttonSize: 20,
```

Comment:
I _believe_ you should also update the version of the store, so that the changes take effect

Reply:
The changes do seem to take effect live and still persist after reload.

Reply:
Nevermind, I just now realized what you meant but I'm also not sure how much I should bump the version by. Am I meant to go with 6.1 or 7? Also, building it locally and running it seemed to work even with my existing 
no-button-size Feishin config.

Reply:
I do see it change, but for zustand persist, I believe the correct approach to make migrations happen is bump the version (to 7)